### PR TITLE
Muazzam chaud/issue411

### PR DESCRIPTION
**Pull request by MuazzamChaud**

### CODE REVIEW

File: src/csvcubedpmd/models/rdf/pmdcat.py [hunk 1]
```diff
@@ -4,16 +4,17 @@
 """
 import rdflib
 from rdflib import Namespace, URIRef, Literal
-from typing import Annotated
+from typing import Annotated, Optional
 from abc import ABC
 from datetime import datetime
 from csvcubedmodels.rdf.triple import Triple, InverseTriple, PropertyStatus
+from csvcubedmodels.rdf.datatypes import MARKDOWN
```
Comment (canwaf) [inline src/csvcubedpmd/models/rdf/pmdcat.py:11]: Is MARKDOWN treated as a namespace, is that why it's in all caps?

File: tests/features/dcatcommandline.feature [hunk 2]
```diff
@@ -36,7 +36,7 @@ Feature: Testing the csvw command group in the CLI
           pmdcat:graph <http://data-graph-uri>;
           pmdcat:metadataGraph <http://catalog-metadata-graph-uri>;
           dct:creator <https://www.gov.uk/government/organisations/hm-revenue-customs>;
-          dct:description "All bulletins provide details on percentage of one litre or less bottles. This information is provided on a yearly basis."@en;
+          pmdcat:markdownDescription "All bulletins provide details on percentage of one litre or less bottles. This information is provided on a yearly basis."^^<https://www.w3.org/ns/iana/media-types/text/markdown#Resource>;
```
Comment (canwaf) [inline tests/features/dcatcommandline.feature:39]: Do we not want to say the Markdown is in English?

**Review by canwaf — APPROVED** [review]
I'm fine with it, but is it necessary to remove the language tag we were providing before to support Markdown?

File: tests/features/dcatcommandline.feature (hunk 2 above)
Comment (MuazzamChaud) [reply tests/features/dcatcommandline.feature:39]: it i sn't possible to set the lang and datatype at the same time.
 ```
>>> g.add((URIRef("http://subj"), URIRef("http://predicate"), Literal("hello world", "en", datatype="http://some-datatype")))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/csvcubed-pmd-venv/lib/python3.10/site-packages/rdflib/term.py", line 553, in __new__
    raise TypeError(
TypeError: A Literal can only have one of lang or datatype, per http://www.w3.org/TR/rdf-concepts/#section-Graph-Literal
```

File: src/csvcubedpmd/models/rdf/pmdcat.py (hunk 1 above)
Comment (robons) [reply src/csvcubedpmd/models/rdf/pmdcat.py:11]: I stuck that there.

**Pull request merged**